bindings/pyroot_experimental/PyROOT/CMakeLists.txt

```diff
@@ -58,4 +58,12 @@ endforeach()
 ROOT_LINKER_LIBRARY(ROOTPython ${sources} LIBRARIES Core Tree cppyy)
 ROOT_INSTALL_HEADERS(inc)
 
+# Disables warnings originating from deprecated register keyword in Python
```

Comment:
Could you add to the commit message that the warnings come from a Python2.7 installation and we observed it with clang 10.0 that checks for C++17 compatibility?

Reply:
`deprecated-register`